sonarts-core/tests/runner/tsrunner.test.ts

```diff
@@ -111,11 +111,13 @@ it("should log incremental reports about analysis", () => {
     loggedOutput += chunk.toString();
     return true;
   });
+  // monkey-patching Math.round to avoid waiting 10s
+  jest.spyOn(Math, "round").mockImplementation(() => 11);
```

Comment:
https://jestjs.io/docs/en/timer-mocks.html

Reply:
If I'm not wrong those timer-mocks are to mock setTimeout or setInterval functions. Since we're not using them anymore I'm not sure we could still benefit from those "timer-mocks"